core/PhysiCell_cell.cpp

```diff
@@ -1399,12 +1399,33 @@ void Cell::ingest_cell( Cell* pCell_to_eat )
 		// absorb the internalized substrates 
 		
 		// multiply by the fraction that is supposed to be ingested (for each substrate) 
+
 		*(pCell_to_eat->internalized_substrates) *= 
 			*(pCell_to_eat->fraction_transferred_when_ingested); // 
-		
+
 		*internalized_substrates += *(pCell_to_eat->internalized_substrates); 
 		static int n_substrates = internalized_substrates->size(); 
 		pCell_to_eat->internalized_substrates->assign( n_substrates , 0.0 ); 	
+
+		// conserved quantitites in custom data aer divided in half
+		// so that each daughter cell gets half of the original ;
+		for( int nn = 0 ; nn < custom_data.variables.size() ; nn++ )
+		{
+			custom_data.variables[nn].conserved_quantity = true; 
```

Comment:
I think we want to delete this line.

Reply:
oof, right.  I think I was forcing it during testing since it wasn't being parsed.

Reply:
did the change, thank you!